features/titletagchange_133937155.feature

```diff
@@ -0,0 +1,27 @@
+Feature: I want to shorten the static part of the title to contain only "Harrow volunteering" instead of "Harrow Community Network | Harrow volunteering"
```

Comment:
we wouldn't usually have the story id in the filename here - also I'm not sure we need a separate feature file here - I think the functional change is actually covered by `features/admin/seo.feature` so we could perhaps remove this one ...